examples/rossmann-store-sales-example.ipynb

```diff
@@ -514,19 +514,19 @@
    "outputs": [],
    "source": [
     "import torch\n",
-    "from nvtabular.torch_dataloader import AsyncTensorBatchDatasetItr, DLDataLoader\n",
+    "from nvtabular.torch_dataloader import TorchAsyncItr, DLDataLoader\n",
```

Comment:
Should we use this for the vanilla pytorch version here as well as the fastai version?

Reply:
you mean use both the torch.utils.dataloader and the DLDataloader...

Reply:
We have two versions of using the torch_dataloader in this notebook now with this PR https://github.com/NVIDIA/NVTabular/pull/147 - one that uses the fastai framework for training and one that only uses pytorch. I think this PR is only updating the fastai version